File: tuts/005-cloudfront-gettingstarted/java/src/main/java/com/example/cloudfront/GettingStartedScenario.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.cloudfront;

import software.amazon.awssdk.services.cloudfront.CloudFrontClient;

public class GettingStartedScenario {
    public static void main(String[] args) {
        try (CloudFrontClient client = CloudFrontClient.builder().build()) {
            CloudFrontWrapper wrapper = new CloudFrontWrapper(client);
            System.out.println("Running CloudFront getting started scenario...");
            // TODO: setup, interact, teardown
            System.out.println("Scenario complete.");
        }
    }
}

File: tuts/005-cloudfront-gettingstarted/kotlin/src/main/kotlin/com/example/cloudfront/GettingStartedScenario.kt
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.cloudfront

import aws.sdk.kotlin.services.cloudfront.CloudFrontClient
import kotlinx.coroutines.runBlocking

fun main() = runBlocking {
    CloudFrontClient { region = "us-east-1" }.use { client ->
        val wrapper = CloudFrontWrapper(client)
        println("Running CloudFront getting started scenario...")
        // TODO: setup, interact, teardown
    }
}
